architecture/README.md

```diff
@@ -180,7 +142,7 @@ on top of the merkledag, such as:
 
 See more in the merkledag spec (TODO).
 
-### [3.4.1 Merkledag Paths](../merkledag)
+## [3.4.1 Merkledag Paths](../ipld)
```

Comment:
Good point. IPLD was merged on top of the MerkleDAG. I think this can be more obvious :)